<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_font_asm">
     <property name="text">
      <string>Font (assembler)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFontComboBox" name="cb_font_asm">
     <property name="currentText">
      <string>Source Code Pro [ADBE]</string>
     </property>
     <property name="fontFilters">
      <set>QFontComboBox::MonospacedFonts</set>
     </property>
     <property name="currentFont">
      <font>
       <family>Source Code Pro [ADBE]</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_compatibility">
     <property name="text">
      <string>Compatibility</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="cb_pnut">
     <property name="text">
      <string>&amp;PNut compatible listing</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_v33mode">
     <property name="text">
      <string>Syntax</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="cb_v33mode">
     <property name="text">
      <string>Use &amp;V33 syntax for PTRA/PTRB index</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbl_file_errors">
     <property name="text">
      <string>FILE errors</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cb_file_errors">
     <property name="text">
      <string>Emit errors on missing &amp;FILEs</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbl_font_dasm">
     <property name="text">
      <string>Font (disassembler)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QFontComboBox" name="cb_font_dasm">
     <property name="currentText">
      <string>Source Code Pro [ADBE]</string>
     </property>
     <property name="fontFilters">
      <set>QFontComboBox::MonospacedFonts</set>
     </property>
     <property name="currentFont">
      <font>
       <family>Source Code Pro [ADBE]</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
